src/index.html

```diff
@@ -16,7 +28,82 @@
       href="./style.css"
     />
   </head>
+  <header>
+    <img
+      src="./images/logo.png"
+      alt="logo.png"
```

Comment:
The 'alt' attribute for the image should be descriptive of the image content, not just the file name. Consider using a more descriptive text like 'Company Logo'.